docs/report/report.tex

```diff
@@ -399,12 +422,175 @@ \subsubsection{Frameworks}
 Neben der Programmiersprache Typescript setzte das Team verschiedene Frameworks ein, um bei der Entwicklung auf bereits existierenden Lösungen mit weniger Aufwand zu entwickeln.
 
```

Comment:
In meinem Kopf ergibt der Satzbau so keinen Sinn?
um auf existierenden Lösungen *aufzubauen*? und *daher* mit weniger Aufwand zu entwicklen?

Reply:
```suggestion
Neben der Programmiersprache Typescript setzte das Team verschiedene Frameworks ein, um mit weniger Aufwand entwickeln zu können.

```